sim/src/core/Simulator.cpp

```diff
@@ -470,34 +460,21 @@ namespace mars {
       case RUNNING:
         // Allow update process to finish -> transition from 2 -> 0 in main loop
         simulationStatus = STOPPING;
-        //fprintf(stderr, "Simulator will be stopped\t");
-        //fflush(stderr);
         stepping_wc.wakeAll();
         break;
       case STOPPING:
-        //fprintf(stderr, "WARNING: Simulator is stopping. Start/Stop Trigger ignored.\t");
-        //fflush(stderr);
         break;
       case STOPPED:
         simulationStatus = RUNNING;
-        //fprintf(stderr, "Simulator has been started\t");
-        //fflush(stderr);
         break;
       case STEPPING:
          simulationStatus = RUNNING;
          stepping_wc.wakeAll();
       default: // UNKNOWN
-        //fprintf(stderr, "Simulator has unknown status\n");
         throw std::exception();
```

Comment:
It would make sense to give at least a hint what the reason for the exception is